floss/render/default.py

```diff
@@ -135,7 +135,7 @@ def render_static_substrings(strings, encoding, offset_len, console, verbose, di
     render_sub_heading(f"FLOSS STATIC STRINGS: {encoding}", len(strings), console, disable_headers)
     for s in strings:
         if verbose == Verbosity.DEFAULT:
-            console.print(s.string)
+            console.print(escape(s.string))
```

Comment:
we'll also want to use `markup=False` here:
https://rich.readthedocs.io/en/stable/reference/console.html#rich.console.Console

```suggestion
            console.print(escape(s.string), markup=False)
```

Reply:
do we need both or does `markup=False` suffice?

Reply:
`markup=False` is sufficient to fix the coloring issue, but it seems like maybe the escape might still be useful for things like newlines?